charts/intel-resource-drivers/values.yaml

```diff
@@ -0,0 +1,155 @@
+gpu:
+  fullnameOverride: ""
+  nameOverride: ""
+  namespaceOverride: "intel-gpu-resource-driver"
+
+  imagePullSecrets: []
+  image:
+    repository: intel
+    name: intel-gpu-resource-driver
+    pullPolicy: IfNotPresent
+    tag: "v0.5.1"
+
+  serviceAccount:
+    create: true
+    annotations: {}
+    name: ""
+    automount: true
+
+  rbac:
+    create: true
+    createAggregateClusterRoles: false
+    pspEnabled: false
+    pspUseAppArmor: false
+
+  # gpu controller
+  controller:
+    tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      - key: node-role.kubernetes.io/control-plane
+        operator: Exists
+        effect: NoSchedule
+    nodeSelector:
+      {}
+      #node-role.kubernetes.io/control-plane: ""
+    affinity: {}
+    containers:
+      controller:
+        securityContext:
+          privileged: false
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop: ["ALL"]
+          readOnlyRootFilesystem: true
+          runAsNonRoot: true
+          runAsUser: 10001
```

Comment:
Should user IDs be configurable?  Or maybe differ between drivers?

Reply:
I noticed that we haven't changed these user IDs in the past. Has there been a new requirement or specific request that prompts this change now?

Reply:
It's more of a nice-to-have, using different users may provide marginal security improvement.

(Or if you meant user IDs for GPU workloads instead of k8s device drivers, reason for that is legacy: https://kubernetes.io/blog/2021/11/09/non-root-containers-and-devices/)

Reply:
I configured this field in the template:
```securityContext:
  {{- toYaml .Values.gpu.controller.containers.controller.securityContext | nindent 10 }}
  
All settings under `securityContext `including `runAsUser` field are customized using the values.yaml file. Is this what you were looking for?

Reply:
I was just commenting about the defaults in values here.   Currently both `gpu` and `gaudi` controllers use the same `10001` user, and I was proposing using different default user IDs for (very) marginal security benefit.